source/renderer/app/assets/images/spinner-dark-big.inline.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="88" height="88" viewBox="0 0 88 88">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 88 88">
```

Comment:
@mchappell do you think `asset/source` loader could've added side-effects to other svgs file? 

![Screen Shot 2022-01-05 at 17 33 52](https://user-images.githubusercontent.com/2846918/148289609-843c4fd2-7067-41b6-9cef-f3d200d16d4c.png)

Reply:
Good spot! Fixed this in latest push